tests/base/src/test/java/org/wildfly/security/auth/server/CustomNameRewriterTest.java

```diff
@@ -0,0 +1,75 @@
+package org.wildfly.security.auth.server;
+
+import org.junit.Test;
+import org.wildfly.security.auth.permission.LoginPermission;
+import org.wildfly.security.auth.realm.FileSystemSecurityRealm;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class CustomNameRewriterTest {
+
+    private static final String BEFORE_USER_NAME = "bob";
+    private static final String AFTER_USER_NAME = "Robert";
```

Comment:
This is very small, but "bob" is all lowercase, and "Robert" has a capital R. Is there a reason for that? 
If not, could we change it to "Bob" instead?